shop/src/main/java/ro/msg/learning/shop/models/Location.java

```diff
@@ -0,0 +1,34 @@
+package ro.msg.learning.shop.models;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Data
+@Table(name = "location")
+@Getter @Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class Location extends  BaseEntity{
+
+    private String name;
+    private String addressCountry;
+    private String addressCity;
+    private String addressCounty;
+
+    private String adressStreetAddress;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "location", cascade = CascadeType.ALL)
+    private List<Revenue> revenueList;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "location", cascade = CascadeType.ALL)
+    private List<Order> orders;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "location", cascade = CascadeType.ALL)
+    private List<Stock> stocks;
```

Comment:
You don't need these lists. See previous comment

Reply:
Deleted every list in a one-to-many relationship.